.github/workflows/ci.yml

```diff
@@ -66,6 +66,31 @@ jobs:
           -DS3_PRESIGNED_PUT_URL="put-url"
       - name: Build System Tests
         run: make -C build/ help | grep system | tr -d '. ' | xargs make -C build/
+  build-check-install:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone This Repo
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Configure CMake build
+        run: |
+          sudo mkdir installed-headers-dir installed-libs-dir
+          curl https://cmake.org/files/v3.2/cmake-3.2.0-Linux-x86_64.tar.gz -o cmake.tar.gz
+          tar -xf cmake.tar.gz
+          mkdir build && cd build
+          ../cmake-3.2.0-Linux-x86_64/bin/cmake .. \
+          -DBUILD_DEMOS=0 -DBUILD_TESTS=0 \
+          -DCSDK_HEADER_INSTALL_PATH="$PWD/../installed-headers-dir" -DCSDK_LIB_INSTALL_PATH="$PWD/../installed-headers-dir"
+      - name: Install libraries and verify install location
+        run: |
+          cd build && sudo make install
+          while IFS="" read -r p || [ -n "$p" ]
```

Comment:
A comment explaining about the of `install_manifest.txt` and the meaning of this loop will be helpful for future reference.